uint8  MOTION_RESET_COMMAND_TYPE_WHEEL = 0
uint8  MOTION_RESET_COMMAND_TYPE_ODOMETRY = 1

uint8 type